File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.MoreEx_JS_Conditional_Stitiments-More Exercises/P05_Pets.js
function pets(input) {

    let daysNumber = Number(input[0]);
    let foodKg = Number(input[1]);
    let dogPortionKg = Number(input[2]);
    let catPortionKg = Number(input[3]);
    let turtlePortionGr = Number(input[4]);

    let neededFood = daysNumber * (dogPortionKg + catPortionKg + turtlePortionGr / 1000);

    if (neededFood > foodKg) {
        console.log(`${Math.ceil(neededFood - foodKg)} more kilos of food are needed.`);
    } else {
        console.log(`${Math.floor(foodKg - neededFood)} kilos of food left.`);
    }
}

pets([5,
    10,
    2.1,
    0.8,
    321
])

File: Java_Basic_Jan_2023_SolutionsOfProblems/P02AdditionalProblems_ConditionalStatements/P05_Pets.java
package P02AdditionalProblems_ConditionalStatements;

import java.util.Scanner;

public class P05_Pets {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int days = Integer.parseInt(scanner.nextLine());
        int foodWeight = Integer.parseInt(scanner.nextLine());
        double dogFood = Double.parseDouble(scanner.nextLine());
        double catFood = Double.parseDouble(scanner.nextLine());
        double turtleFood = Double.parseDouble(scanner.nextLine()) / 1000;
        double totalPetsFood = (dogFood + catFood + turtleFood) * days;

        if (totalPetsFood <= foodWeight)
            System.out.printf("%.0f kilos of food left.", Math.floor(foodWeight - totalPetsFood));
        else
            System.out.printf("%.0f more kilos of food are needed.", Math.ceil(totalPetsFood - foodWeight));
    }
}
